src/pybind/mgr/dashboard_v2/tests/test_dashboard.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import
+
+from cherrypy.lib.sessions import RamSession
+
+import mock
+
+from .helper import ControllerTestCase
+from ..controllers.auth import Auth
+from ..tools import Session
+
+
+class DashboardlTest(ControllerTestCase):
+
+    @classmethod
+    def setup_test(cls):
+        cls._mgr_module.set_config('username', 'admin')
+        cls._mgr_module.set_config('password', Auth.password_hash('admin'))
+
+    @mock.patch('mgr_module.MgrModule.get')
+    def test_toplevel(self, osd_map_mock):
+        sess_mock = RamSession()
+        with mock.patch('cherrypy.session', sess_mock, create=True):
```

Comment:
Can't we create some kind of decorator or helper function to preform the login, so that we can use it in other tests?
Almost all controllers will require authentication and therefore we should have some easy way to preform authenticated requests.